File: VB/DXApplication189/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.LookAndFeel
Imports DevExpress.Skins
Imports DevExpress.Utils.Drawing
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraEditors
Imports DevExpress.XtraGrid.Views.Tile
Imports DevExpress.XtraGrid.Views.Tile.ViewInfo

Namespace DXApplication189
    Partial Public Class Form1
        Inherits RibbonForm

        Public Sub New()
            InitializeComponent()
            Dim helper As New TileViewDrawButtonHelper(tileView1) With {.ButtonCaption = "Add task"}
            AddHandler helper.GroupButtonClick, AddressOf Helper_GroupButtonClick
        End Sub

        Private Sub Helper_GroupButtonClick(ByVal sender As Object, ByVal args As GroupButtonClickEventArgs)
            MessageBox.Show(args.Group.Group.Text)
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            ' TODO: This line of code loads data into the 'nwindDataSet.Customers' table. You can move, or remove it, as needed.
            Me.customersTableAdapter.Fill(Me.nwindDataSet.Customers)
        End Sub
    End Class


End Namespace

File: CS/DXApplication189/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.LookAndFeel;
using DevExpress.Skins;
using DevExpress.Utils.Drawing;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraEditors;
using DevExpress.XtraGrid.Views.Tile;
using DevExpress.XtraGrid.Views.Tile.ViewInfo;

namespace DXApplication189 {
    public partial class Form1 : RibbonForm {
        public Form1() {
            InitializeComponent();
            TileViewDrawButtonHelper helper = new TileViewDrawButtonHelper(tileView1) { ButtonCaption = "Add task"};
            helper.GroupButtonClick += Helper_GroupButtonClick;
        }

        private void Helper_GroupButtonClick(object sender, GroupButtonClickEventArgs args) {
            MessageBox.Show(args.Group.Group.Text);
        }

        private void Form1_Load(object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'nwindDataSet.Customers' table. You can move, or remove it, as needed.
            this.customersTableAdapter.Fill(this.nwindDataSet.Customers);
        }
    }


}
